trinity/nodes/base.py

```diff
@@ -82,3 +117,20 @@ def start_loop(loop):
         thread.start()
 
         return new_loop
+
+
+def create_db_manager(ipc_path: str) -> BaseManager:
+    class DBManager(BaseManager):
+        pass
+
+    # Typeshed definitions for multiprocessing.managers is incomplete, so ignore them for now:
+    # https://github.com/python/typeshed/blob/85a788dbcaa5e9e9a62e55f15d44530cd28ba830/stdlib/3/multiprocessing/managers.pyi#L3
+    DBManager.register('get_db', proxytype=DBProxy)  # type: ignore
+    DBManager.register('get_chaindb', proxytype=ChainDBProxy)  # type: ignore
+    DBManager.register('get_chain', proxytype=ChainProxy)  # type: ignore
+    DBManager.register('get_headerdb', proxytype=AsyncHeaderDBProxy)  # type: ignore
+    DBManager.register('get_header_chain', proxytype=AsyncHeaderChainProxy)  # type: ignore
+
+    manager = DBManager(address=ipc_path)  # type: ignore
+    manager.connect()  # type: ignore
```

Comment:
Maybe the `connect()` should be moved out of this function, or rename to `create_and_connect_db_manager()` or `get_db_manager` or `construct_db_manager`.  Something about `create` and the class being named `DBManager` makes me think it's just going to return the class.

Reply:
`launch_db_manager()`?

Reply:
Historically I feel like I always *wish* I'd separated `construct -> run` into individual units of functionality.

Reply:
Yeah, there are a number of things that I would like to wrap up into a `BaseService`. DBManager might be one of them. Then it should have a clear: init/run/stop cycle.